app/assets/javascripts/discourse/components/notification-consent-banner.js.es6

```diff
@@ -0,0 +1,45 @@
+import { default as computed } from 'ember-addons/ember-computed-decorators';
+
+import {
+  keyValueStore as pushNotificationKeyValueStore
+} from 'discourse/lib/push-notifications';
+
+import { default as DesktopNotificationConfig } from 'discourse/components/desktop-notification-config';
+
+const userDismissedPromptKey = "dismissed-prompt";
+
+export default DesktopNotificationConfig.extend({
+  @computed
+  bannerDismissed: {
+    set(value) {
+      pushNotificationKeyValueStore.setItem(userDismissedPromptKey, value);
+      return pushNotificationKeyValueStore.getItem(userDismissedPromptKey);
+    },
+    get() {
+      return pushNotificationKeyValueStore.getItem(userDismissedPromptKey);
+    }
+  },
+
+  @computed("isNotSupported", "isEnabled", "bannerDismissed")
```

Comment:
add `currentUser.reply_count`  and `currentUser.topic_count`  to computed here, so we show the banner after the first post without requiring a full refresh